packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -169,100 +186,122 @@ const App = (props) => {
             >
                 <CategoriesProvider categories={allCategories}>
                     <CurrencyProvider currency={currency}>
-                        <Seo>
-                            <meta name="theme-color" content={THEME_COLOR} />
-                            <meta name="apple-mobile-web-app-title" content={DEFAULT_SITE_TITLE} />
-                            <link
-                                rel="apple-touch-icon"
-                                href={getAssetUrl('static/img/global/apple-touch-icon.png')}
-                            />
-                            <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
-
-                            {/* Urls for all localized versions of this page (including current page)
+                        <SiteProvider site={site}>
+                            <LocaleProvider locale={locale}>
```

Comment:
Hierarchically, SiteProvider => LocaleProvider => CurrencyProvider => CategoryProvider

Reply:
I see... There can be multiple sites in the app. Each site can have multiple locales, and each locale has an associated currency 👍